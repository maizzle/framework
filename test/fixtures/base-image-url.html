<img src="test.jpg" alt="">

<div style="background-image: url('test.jpg')">Test</div>

<div style="background-image: url(test.jpg)">Test</div>

<table background="test.jpg">
  <tr>
    <td>Test</td>
  </tr>
</table>
