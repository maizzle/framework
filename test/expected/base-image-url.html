<img src="https://example.com/test.jpg" alt="">

<div style="background-image: url('https://example.com/test.jpg')">Test</div>

<div style="background-image: url(https://example.com/test.jpg)">Test</div>

<table background="https://example.com/test.jpg" cellpadding="0" cellspacing="0" role="presentation">
  <tr>
    <td>Test</td>
  </tr>
</table>
